script.js

```diff
@@ -0,0 +1,96 @@
+// navigation Script
+const menuIcon = document.querySelector('.fa-bars');
+const mobileMenuList = document.querySelector('.mobile-menu-pane');
+const mobileHeader = document.querySelector('.intro');
+const bodyS = document.querySelector('.body');
+
+menuIcon.addEventListener('click', () => {
+  mobileHeader.classList.toggle('header-bg');
+  menuIcon.classList.toggle('fa-xmark');
+  mobileMenuList.classList.toggle('opened');
+  bodyS.classList.toggle('stopScroll');
+});
+
+function removeStuffs() {
+  mobileMenuList.classList.remove('opened');
+  menuIcon.classList.remove('fa-xmark');
+  mobileHeader.classList.remove('header-bg');
+  bodyS.classList.remove('stopScroll');
+}
+
+// Speaker object prototype
+function Speaker(fullname, title, desc, img) {
+  this.fullname = fullname;
+  this.title = title;
+  this.desc = desc;
+  this.img = img;
+}
+
+function getSpeakers() {
+// create array to store projects
+  let speakers = [];
+  const speakerInfo1 = new Speaker('Dr Mahamudu Bawumia', ' Associate Professor of Innovation and Entrepreneurship', 'A Design Strategist and a Tech Enthusiast', 'img/alhaji-mahamudu-bawumia.jpeg');
+  const speakerInfo2 = new Speaker('Hon. Ursula Owusu', 'Minister of Communication and Digital Economy', 'A Design Strategist and a Tech Enthusiast', 'img/ursula-owusu.hpeg.jpg');
+  const speakerInfo3 = new Speaker('Hon. Dr. Kwaku Afriyie', 'Minister, Environment, Science, Technology and Innovation (MESTI)', 'A Design Strategist and a Tech Enthusiast', 'img/hon-kwaku-afriyie.jpeg');
+  const speakerInfo4 = new Speaker('Dr Gordon Adomdza', 'Associate Professor of Innovation and Entrepreneurship', 'A Design Strategist and a Tech Enthusiast', 'img/gordon.jpg');
+  const speakerInfo5 = new Speaker('Dr Gordon Adomdza', 'Associate Professor of Innovation and Entrepreneurship', 'A Design Strategist and a Tech Enthusiast', 'img/mypic.jpg');
+  const speakerInfo6 = new Speaker('Dr Gordon Adomdza', 'Associate Professor of Innovation and Entrepreneurship', 'A Design Strategist and a Tech Enthusiast', 'img/AbiOla.jpeg');
+  speakers = [speakerInfo1, speakerInfo2, speakerInfo3, speakerInfo4, speakerInfo5, speakerInfo6];
+  return speakers;
+}
+
+function prepareSpeaker(speakers) {
+  const speakerSection = document.getElementById('speakers');
+  const buttomDiv = document.createElement('div');
+  const bottomP = document.createElement('p');
+  bottomP.innerHTML = 'MORE';
```

Comment:
[OPTIONAL] The `more button` is supposed to display the other speakers that are hidden upon clicking such that when all speakers are displayed it should disappear. Currently it is not working like. Kindly check and fix it to make your project look more proffesional and up to standard :pray: 

Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-19 08-42-42](https://user-images.githubusercontent.com/65233860/196662182-84764096-4171-4f4a-a0d3-8a94ed087130.png)  |  ![Screenshot from 2022-10-19 09-52-21](https://user-images.githubusercontent.com/65233860/196662199-e84f9ab8-c6e6-43c9-82ec-28df892e7162.png)